src/Passable.js

```diff
@@ -40,21 +40,15 @@ class Passable {
             return;
         }
 
-        const lastIndex: number = args.length - 1;
-        let callback: Function;
+        this.res.initFieldCounters(fieldName);
 
-        if (typeof args[lastIndex] === 'function') {
-            callback = args[lastIndex];
-        } else {
+        if (typeof test !== 'function') {
             return;
         }
 
-        this.res.initFieldCounters(fieldName);
-
-        const isValid: boolean = testRunner(callback);
+        const isValid: boolean = testRunner(test);
 
         if (!isValid) {
-            const severity: Severity = lastIndex !== 0 ? args[0] : FAIL;
             this.res.fail(fieldName, statement, severity);
         }
```

Comment:
in fact you can do `testRunner(test) && this.res.fail(fieldName, statement, severity);`